ch_tools/chadmin/cli/dictionary_group.py

```diff
@@ -52,3 +56,12 @@ def _full_name(dictionary: Any) -> str:
         return f"`{db_name}`.`{dict_name}`"
 
     return f"`{dict_name}`"
+
+
+@dictionary_group.command("migrate")
+@pass_context
+def migrate_command(ctx: Context) -> None:
+    """
+    Migrate XML-dictionaries to DDL.
```

Comment:
Let's add `dry-run` option to be able to run command and see it's output without actually migrating dictionaries